core/src/progress_map.rs

```diff
@@ -168,7 +168,7 @@ impl ReplaySlotStats {
             ("accumulated_units", total_units, i64),
             ("count", total_count, i64),
             ("errored_units", total_errored_units, i64),
-            ("count", total_errored_count, i64)
+            ("errored_count", total_errored_count, i64)
```

Comment:
Just an FYI, both fields will show up in chronograf until the last "count" falls off retention